frontend/app/services/MemberService.scala

```diff
@@ -305,7 +305,7 @@ class MemberService(identityService: IdentityService,
 
   override def getUsageCountWithinTerm(subscription: Subscription, unitOfMeasure: String): Future[Option[Int]] = {
     val features = subscription.features
-    val startDate = subscription.startDate.toDateTimeAtCurrentTime()
+    val startDate = subscription.startDate.toDateTimeAtStartOfDay(DateTimeZone.forID("America/Los_Angeles"))
```

Comment:
Should `subscription.startDate` have changed to `subscription.termStartDate` with this pull request? The PR description makes it sound like it should:

```
By using the subscription start date rather than the term start date we would not have reset allocations
each year, had we been picking them up at all
```